content/events-api/migrate-to-event-platform.md

```diff
@@ -0,0 +1,98 @@
+To create a subscriber, you need to make a **POST** request to `https://event.prd.sdk.akeneo.cloud/api/v1/subscribers`, including three headers and a **JSON body**.
+
+The required headers are:
+
+- `x-pim-client-id`: The Connection client ID.
+- `x-pim-url`: The URL of the relevant PIM, starting with `https://`.
+- `x-pim-token`: The access token (the same token you send with the `Authorization: Bearer …` header when using the PIM REST API).
+
+The body should contain a single object with a `name` property and a `contact` sub-object, which includes a `technical_email` property. Here’s an example:
+
+```json
+
+{
+    "name": "awesome connection subscriber",
+    "contact": {
+        "technical_email": "valid.email@example.com"
+    }
+}
+```
+
+The response will be a JSON object containing an `id`, which you will need to create your subscription(s).
+
+### Create a subscription
+To replicate a system similar to what you had with the Event API (with a single destination), you will need to create one subscription.
+
+As with the subscriber, you can use Postman or another tool like Curl. The URL to call is `https://event.prd.sdk.akeneo.cloud/api/v1/subscribers/{subscriber_id}/subscriptions`. The headers remain the same as before.
+
+For more details, refer to the documentation for the endpoint [here](https://storage.googleapis.com/akecld-prd-sdk-aep-prd-api-assets/openapi_specification.html#tag/Subscription/operation/create_subscription). To summarize, here’s an example of the payload you need to send:
+
+```json
+
+{
+    "source": "pim",
+    "subject": "https://pim.url",
+    "events": [
+        "com.akeneo.pim.v1.product.created",
+        "com.akeneo.pim.v1.product.updated.delta",
+        "com.akeneo.pim.v1.product.deleted",
+        "com.akeneo.pim.v1.product-model.created",
+        "com.akeneo.pim.v1.product-model.updated.delta",
+        "com.akeneo.pim.v1.product-model.deleted"
+    ],
+    "type": "https",
+    "config": {
+        "url": "https://destination.url",
+        "secret": {
+            "primary": "averysecretprimarysecret",
+            "secondary": "can be empty, only used to ease secret rotation"
+        }
+    }
+}
+```
+
+This example includes all the events necessary to maintain a similar system to what you had before. You can choose fewer or additional events as needed. A complete list of available events can be found [here](https://api.akeneo.com/event-platform/available-events.html#available-events).
+
+Now, all relevant events should be sent. To test the setup, simply create, modify, or delete a product and check your destination to see if you receive the corresponding event.
+
+
+## Development adjustments
+### Product UUID and removal of identifier
+In the previous Event API, you had the option to send product information using either UUID or identifier. With the Event Platform, only the UUID is transmitted. If you were using product identifiers, you will need to transition to using UUIDs.
```

Comment:
```suggestion
In the previous Event API, you had the option to send product information using either UUID or identifier. With the Event Platform, only the UUID is transmitted. If you were using product identifiers, you will need to transition to using UUIDs.
```
"transition" is not a verb, so you should change this word. In my opinion, you can remove the sentence. What do you think?

Reply:
Are you sure ?
![image](https://github.com/user-attachments/assets/766e6f1b-efef-45c4-afb4-f8e9f5302a0d)
Reverso propose it too. But may be it's better to use "make the transition" ?

Reply:
Oh yes, I learnt something!